documentation/docs/04-hooks.md

```diff
@@ -60,6 +60,8 @@ export async function handle({ request, resolve }) {
 }
 ```
 
+> `handle` runs when a page is rendered. If a page is prerendered, `handle` will not run again when the user requests the page. If you would like to run some functionality on each request, see the documentation for your [adapter](#adapters).
+
```

Comment:
```suggestion
```

Reply:
I think we still need the first sentence here. It's not at all obvious from the text above that `handle` doesn't run on prerendered pages. I'm fine with dropping the second sentence though

Reply:
Oh, nevermind. I missed that you added a callout above